Meaningful message instead of blank field when session is not scheduled.

Anupam-dagar opened the pull request:
<!-- 
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

This commit adds a "Session not scheduled yet." message to the session
card when session is not scheduled.


#### Changes proposed in this pull request:

- Changed the if condition for the scheduler block.
- Added an else block to display the message.


<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes: #2105 #2110 

![sessions - test - open event 04-02-2019 17-45-30](https://user-images.githubusercontent.com/21174572/52207858-be105600-28a4-11e9-80dc-af0d49788dd8.png)

Regarding the alignment in the image, it will get resolved automatically once #2104 is merged.

CosmicCoder96 commented on app/templates/components/public/session-item.hbs:
```diff
@@ -20,9 +20,11 @@
         </div>
         <div class="ui right aligned column four wide less left padding">
-          {{#if session.startsAt}}
+          {{#if session.microlocation.name}}
```
Leave this be, rest is fine.

Anupam-dagar replied:
I guess by default there is a time set for the sessions, so by keeping `startsAt` in the if condition, it will always get true and the block will execute. That's why I shifted to microlocation name since on the session getting scheduled, the session will have a location and a time slot for sure.

CosmicCoder96 replied:
actually, for unscheduled sessions it is supposed to be null, seems weird that they will have a startTime by default, maybe open an issue for that too. It _should_ be null.

Anupam-dagar replied:
Opened #2110 to set the unscheduled sessions start and end time to null.

CosmicCoder96 commented on app/templates/components/public/session-item.hbs:
```diff
@@ -23,6 +23,8 @@
             <div class=""><i class="marker icon"></i>{{session.microlocation.name}}</div>
             <div class=""><i class="wait icon"></i>{{moment-format session.startsAt 'hh:mm a / DD-MM-YYYY'}}</div>
+          {{else}}
+            <div>{{t 'Session not scheduled yet.' }}</div>
```
No empty div's please, they serve no purpose.

Anupam-dagar commented:
@CosmicCoder96 I have removed the empty div.
I have added another commit as a fix for the issue #2110 which makes the default value for all the submitted session start and end time to `null`.
